apps/prairielearn/src/pages/instructorGradebook/instructorGradebook.ts

```diff
@@ -53,27 +69,13 @@ router.get(
     if (!res.locals.authz_data.has_course_instance_permission_view) {
       throw new HttpStatusError(403, 'Access denied (must be a student data viewer)');
     }
-    const result = await sqldb.queryAsync(sql.user_scores, {
-      course_id: res.locals.course.id,
-      course_instance_id: res.locals.course_instance.id,
-    });
-
-    res.locals.user_scores_data = _.map(result.rows, function (row) {
-      const scores = {
-        user_id: row.user_id,
-        uid: _.escape(row.uid),
-        uin: _.escape(row.uin ?? ''),
-        user_name: _.escape(row.user_name ?? ''),
-        role: row.role,
-      };
-      row.scores.forEach(function (score) {
-        scores[`score_${score.assessment_id}`] = score.score_perc;
-        scores[`score_${score.assessment_id}_ai_id`] = score.assessment_instance_id;
-        scores[`score_${score.assessment_id}_other`] = _.map(score.uid_other_users_group, _.escape);
```

Comment:
This was changed to use columns like `scores.${assessment_id}.score_perc`, which is supported by bootstrap-table, and then retrieving the other properties in the formatter itself.